delocate/tests/env_tools.py

```diff
@@ -24,3 +24,17 @@ def TempDirWithoutEnvVars(*env_vars):
             else:
                 if var in os.environ:
                     del os.environ[var]
+
+
+@contextmanager
+def Environment(**env):
```

Comment:
Don't name this like a class, add annotations, and a docstring, maybe also make it private.
```suggestion
def _scope_env(**env: str) -> Iterator[None]:
    """Add the provided environment variables for the duration of this context."""
```

Reply:
> Don't name this like a class,

Done. I named the new context manager like a class precisely because of the other context manager function in the same file.

Reply:
I missed the rename suggestion. I'll push a fix.